openedx/core/lib/token_utils.py

```diff
@@ -94,8 +98,17 @@ def attach_profile_claim(self, payload):
         except UserProfile.DoesNotExist:
             name = None
 
+        language = None
+        try:
+            user_preference = self.user.preferences.get(key='pref-lang')
+            language = user_preference.get_all_preferences(self.user).get('pref-lang')
+        except UserPreference.DoesNotExist:
+            log.debug("UserPreference for User: '{}' does not exist.".format(self.user))
```

Comment:
Include the key so that we don't have to hunt in the code to determine which preference was does not exist.